packages/react/src/runtime/style.tsx

```diff
@@ -43,7 +43,8 @@ export default function Style(props: StyleProps): JSX.Element | null {
       }
 
       return (
-        <style data-cmpld nonce={props.nonce}>
+        // set data-cmpld attribute to "s" for SSR
+        <style data-cmpld="s" nonce={props.nonce}>
```

Comment:
This obviously adds a few bytes here but when there are hundreds of SSR'd style tags on the DOM it's actually fewer bytes because React turns this into ```data-cmpld="true"``` so it's actually 3 less bytes per style tag.